10.9-libcxx/stable/main/finkinfo/libs/perlmods/yaml-tiny-pm.info

```diff
@@ -13,8 +13,7 @@ BuildDepends: <<
 	perl%type_pkg[systemperl]-core
 <<
 Source: mirror:cpan:modules/by-module/YAML/YAML-Tiny-%v.tar.gz
-Source-MD5: 9fd7940c05b025d6be08217c9a5061b6
-DefaultScript: ModuleBuild
```

Comment:
Now that it's using Makefile.PL not Build.PL and I don't see any Module::Build residue in Makefile.PL or META.yml, can it drop ```BuildDepends:module-build-tiny-pmXXX``` and therefore drop ```BuildDepends:perlXXX-core``` and therefore drop the whole systemperl/platform-dependent varianting (unfying yaml-tiny-pm-10.9.info back into yaml-tiny-pm.info)?